controllers/meetings.js

```diff
@@ -115,9 +99,9 @@ exports.update = function(req, res) {
 };
 // this lists all meetings - should have some sort of filtering available.
 exports.list = function(req, res) {
-	Meeting.find({participants: req.user._id}, function(err, meetings) {
+  Meeting.find({participants: req.user._id}).populate('admin participants', 'name -_id').populate('availability', 'name').exec(function(err, meetings){
 		if (err) {
```

Comment:
the second call looks like it's calling populate to the availability field, and filling out the name property. There is no name property on the availability ... what does this line do?